src/qt/nametablemodel.h

```diff
@@ -0,0 +1,84 @@
+#ifndef NAMETABLEMODEL_H
+#define NAMETABLEMODEL_H
+
+#include "bitcoinunits.h"
+
+#include <QAbstractTableModel>
+#include <QStringList>
+
+class PlatformStyle;
+class NameTablePriv;
+class CWallet;
+class WalletModel;
+
+/**
+   Qt model for "Manage Names" page.
+ */
+class NameTableModel : public QAbstractTableModel
+{
+    Q_OBJECT
+
+public:
+    explicit NameTableModel(const PlatformStyle *platformStyle, CWallet* wallet, WalletModel *parent = 0);
+    //explicit NameTableModel(CWallet *wallet, WalletModel *parent = 0);
```

Comment:
Please remove or clarify why the code is still relevant (I won't comment that anymore below).